scene/gui/popup_menu.h

```diff
@@ -131,6 +131,11 @@ class PopupMenu : public Popup {
 	ScrollContainer *scroll_container = nullptr;
 	Control *control = nullptr;
 
+	const int DEFAULT_GAMEPAD_EVENT_REPEAT_RATE = 20;
+	const float DEFAULT_GAMEPAD_EVENT_DELAY_MS = 0.5;
+	float gamepad_event_repeat_rate_ms = 0.25;
```

Comment:
This is some weird variable. It's not a constant, but the constructor overwrites the default value, which is then never changed anymore.

Reply:
By weird I mean `gamepad_event_repeat_rate_ms` is never changed, so it could be a constant too.
Also the primitive types *should* be initialized in the header. You can use `0.0` if the default is not important.

Reply:
I get it. It should be easier to understand now

```
	const float DEFAULT_GAMEPAD_EVENT_DELAY_MS = 0.5;
	const float GAMEPAD_EVENT_REPEAT_RATE_MS = 1.0 / 20;
	float gamepad_event_delay_ms = DEFAULT_GAMEPAD_EVENT_DELAY_MS;
```